Add Check in views

casassg opened the pull request:
Check #21 for details on this

rgascons commented on app/settings.py:
```diff
@@ -24,7 +24,8 @@
 
-ALLOWED_HOSTS = ['localhost', 'hackupc.com', 'my.hackupc.com']
+
+ALLOWED_HOSTS = ['localhost', 'hackupc.com', 'my.hackupc.com', '127.0.0.1']
```
what's the reason of adding 127.0.0.1 as a host?

casassg replied:
I guess @alvarogirona wanted to use it as host to try production mode? Idk, didn't add it myself I always use localhost

rgascons commented:
- Caps in all @casassg commits
- What exactly commit 2dd227e is? Is this a merge commit, or a rebase one? Either way shouldn't be there. If it's too much hassle leave it as is, but take it into account for future PRs

zakum42 commented:
What exactly commit 2dd227e is? Is this a merge commit, or a rebase one? Either way shouldn't be there. If it's too much hassle leave it as is, but take it into account for future PRs

zakum42 commented:
Also, do all commits build?

casassg commented:
@zakum42 

- I've always been using non-caps (easier, faster and same result). If you tell me why I should switch I will, it's just a habit I have had for a while, and nobody complained.
-  This commit was me fixing stuff and suddenly realizing that I did too many stuff. It's not a merge or rebase, it's a refactor commit (as it's name says). Therefore it has a lot of changes. I should have done it separately. It includes bug fixes (code was not compiling correctly before) + style checks (follow PEP08 pls) + refactor of views code (improve readability) + move templates to new folder (for some reason github is saying that I deleted some code and added new files)

rgascons commented:
@casassg 

- Caps or no-caps, not both :)
- k

casassg commented:
I always go by no-caps. Also mark as approved if good pls :D

On Sat, Feb 18, 2017, 08:17 Ricard Gascons <notifications@github.com> wrote:

> @casassg <https://github.com/casassg>
>
>    - Caps or no-caps, not both :)
>    - k
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hackupc/backend/pull/71#issuecomment-280851860>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AGl6TfAdon0z19FTW3FLA1c0Z-Ppf7uhks5rdwuggaJpZM4MDssG>
> .
>
